src/devices/onokom.ts

```diff
@@ -0,0 +1,2093 @@
+import {Zcl} from "zigbee-herdsman";
+import * as m from "../lib/modernExtend";
+import * as reporting from "../lib/reporting";
+import type {DefinitionWithExtend} from "../lib/types";
+
+const manufacturerCode = 0x4703;
+
+export const definitions: DefinitionWithExtend[] = [
+    {
+        zigbeeModel: ["AUX-1-ZB-S"],
+        model: "AUX-1-ZB-S",
+        vendor: "ONOKOM",
+        description: "ONOKOM AIR AUX-1-MB-B (Adapter for household and semi-industrial AUX systems)",
+        ota: true,
+        extend: [
+            m.enumLookup({
+                name: "AC connected",
```

Comment:
```suggestion
                name: "ac_connected",
```

Make all of these `snake_case` (z2m convention)